packages/kde

```diff
@@ -529,7 +529,7 @@ py38-toml
 py38-zipp
 pydbus-common
 pygobject3-common
-python38
+python38_3,2
```

Comment:
Change that to `python` it will install the latest Python 3.

Reply:
Ok. Thanks.